BitwardenShared/UI/Auth/StartRegistration/StartRegistrationProcessor.swift

```diff
@@ -229,5 +234,16 @@ extension StartRegistrationProcessor: RegionDelegate {
         state.region = region
         state.showReceiveMarketingToggle = state.region != .selfHosted
         await delegate?.didChangeRegion()
+
+        if await !services.configService.getFeatureFlag(
+            .emailVerification,
+            defaultValue: false,
+            forceRefresh: true,
+            isPreAuth: true
+        ) {
+            // If email verification isn't enabled in the selected environment, switch to the
+            // legacy create account flow.
+            delegate?.switchToLegacyCreateAccountFlow()
+        }
```

Comment:
🤔 How does this behave on really slow internet connection scenarios?

Reply:
It won't switch you back to the legacy flow until the request completes, but it doesn't block the UI or anything. Hopefully this is just a temporary change until the legacy flow is removed.

Reply:
No, not until the request times out at least. I added a check to only notify the delegate if the view is still visible.